group_vars/control_room_all.yml

```diff
@@ -0,0 +1,25 @@
+# === NFS Client
+
+nfs_client_packages:
+  - glusterfs-client
+  - glusterfs-common
+  - glusterfs-server
+
+nfsclient_autofs_glusterfs_backup_server: lnls452-linux
+nfsclient_autofs_glusterfs_log_file: /var/log/gluster.log
+
+# autofs: auto.sirius
+nfsclient_autofs:
+  - |
+    /home/opis                         -fstype=nfs,ro {{ nfs_server_control_opi }}:/home/sirius
+  - |
+    {{ nfsclient_autofs_sirius_epics_screens }}   -fstype=nfs,ro {{ nfs_server_control_opi }}:{{ nfsserver_sirius_epics_screens }}
+  - |
+    /home/sirius/repos                 -fstype=nfs,ro {{ nfs_server_control_room }}:/home/nfs-shared/repos-lnls-sirius
+  - |
+    /home/sirius/mounts/lost-and-found -fstype=nfs,rw,v3 {{ nfs_server_control_room }}:/home/nfs-shared/lost-and-found
+  - |
+    /home/sirius/mounts/screens-iocs   -fstype=nfs,rw,v3 {{ nfs_server_control_room }}:/home/nfs-shared/screens-iocs
+  - |
+    /home/sirius/shared                -fstype=glusterfs,backupvolfile-server={{ nfsclient_autofs_glusterfs_backup_server }},log-level=WARNING,log-file={{ nfsclient_autofs_glusterfs_log_file }},rw  localhost:/shared   
```

Comment:
Mounting the glusterfs volume using autofs as it is more reliable than `/etc/fstab`